### PR TITLE
Added Motor Rotational Velocity

### PR DESCRIPTION
**Pull request by pranay2811**
Added telemetry fields:
- motor_rotationalvelocity_sensor
- motor_rotationalvelocity_setpoint

Added Abstract Type:
- MRVC

### CODE REVIEW

**Comment by pranay2811** [comment]
@shambergoldstein Ready for your review

**Comment by pranay2811** [comment]
@shambergoldstein How do I resolve the conflicts in this branch?

File: ontology/yaml/resources/fields/telemetry_fields.yaml
Comment (shambergoldstein) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1]: please add spaces after "-"

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/fields/telemetry_fields.yaml
Comment (pranay2811) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:1]: done

**Review by shambergoldstein — APPROVED** [review]

File: ontology/yaml/resources/fields/telemetry_fields.yaml
Comment (shambergoldstein) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1]: typo on these; "fixed_mind", please fix

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/fields/telemetry_fields.yaml
Comment (pranay2811) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:1]: Fixed it.

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**